web/themes/custom/usagov/templates/field--node--body--wizard-step.html.twig

```diff
@@ -1,11 +1,13 @@
-{% set test = element['#object'].body[0].value|raw %}
+{% set answer = element['#object'].body[0].value|raw %}
```

Comment:
Oops! This is still referred to as "test" on line 3, so the final page is coming up without any content for me.